### PR TITLE
fix(refresh): make sure tooltip appears on disabled button state

### PR DESCRIPTION
**Pull request by yakkomajuri**
Fixing a regression from the reversal back to Antd from Lemon in #14178 

Antd has this bug:

https://github.com/ant-design/ant-design/issues/9581

### CODE REVIEW

File: frontend/src/scenes/insights/ComputationTimeWithRefresh.tsx [hunk 1]
```diff
@@ -11,20 +11,24 @@ export function ComputationTimeWithRefresh(): JSX.Element | null {
 
     usePeriodicRerender(15000)
 
+    console.log(insightRefreshButtonDisabledReason)
```
Comment (Twixes) [inline frontend/src/scenes/insights/ComputationTimeWithRefresh.tsx]: ❗

**Review by Twixes — APPROVED** [review]

**Comment by yakkomajuri** [comment]
I'd like to express in words again that I'm having a tough time again with visual regression tests 😂

**Comment by yakkomajuri** [comment]
current snapshots didn't pass, but the commit I pushed for updating them didn't pass either 🤷‍♂️

**Comment by yakkomajuri** [comment]
welp, that accidentally went in. I had just done the other snapshots too

**Pull request merged**